dev/breeze/src/airflow_breeze/commands/testing_commands_config.py

```diff
@@ -211,6 +211,7 @@
         TEST_OPTIONS_DB,
         TEST_ENVIRONMENT_DB,
         TEST_ADVANCED_FLAGS,
+        TEST_ADVANCED_FLAGS_FOR_INSTALLATION,
```

Comment:
BTW, @vincbeck  -> I think that was the reason @o-nikolas  added it - because those "advanced" flags are adding all the flags to config and breeze expects them to be added there.

Reply:
I see, makes sense. But, as you mentioned, we should pass them to `ShellParams`, otherwise they woll not do anything, which can be really frustrating for users.

Reply:
Yupp, this basically. This gives you access to the use-airflow-version param, but it brings some others and if you don't add them to the signature of the system_tests method then Click complains and fails.

I don't think the others are as relevant for system tests but I'll for sure plumb them through (just pushed a commit for it)

Reply:
nice!